atomic_reactor/schemas/config.json

```diff
@@ -449,7 +449,6 @@
                 "description": "Registry url",
                 "type": "string"
             },
-            "organization": {"$ref": "#/definitions/organization"},
```

Comment:
What's the reason for this change?

Reply:
organization now is set via the `registries_organization` setting.

Reply:
we are using genera 'registries_organization'